apps/wallet-mobile/src/features/Receive/common/mocks.ts

```diff
@@ -0,0 +1,12 @@
+export const mocks = {
+    stakinghash: "absd1h2783g12392bbmmr9uvhmelw2fk3qvd20ja0a6q44x455c6mm8960tqgvtuy2v7ev5a7wzd7l252j5ztk9y2s4mveccadjuq2evgdg",
+    spendinghash: "bbywyasdindsadne03aagdmmr9uvhmelw2iad9uh3892y3891292135c6mm8960tqgvtuy2v7ev5a7wzd7l252j5ztk9y2s4mveccadjuq2evgdg",
+    specificAddressAmount: "web+cardano:addr_test1qrpycewkttfct7tcxud75etqqz57hgnul7tsh47lt4z53zys36la9r2zgza4456kwnhrkp5dl3t3d7a24ktrzj22en3qwtez5g?amount=500",
+    shareFileName: "Share-address",
+    address: "addr1qxstpe4lqp9y04s094twu6p63ht2rdhu9sdd40s5n0leppecjfk0ywkc7w8rxrdxp2xvkk9tewfr39n5naexzryg3c7sxmnh0t",
+    usedAddressDate: '02/02/2024, 14:32',
+    yoroiZendesk: 'Yoroi Zendesk',
+    addressList: [{ isUsed: true, loading: false }],
```

Comment:
is the lint passing ? it should not, this repo doesn't allow spaces between brackets/para...

Reply:
◀️